Multiplatform/Collections/Artists/ArtistList.swift

```diff
@@ -11,13 +11,78 @@ import AFBase
 struct ArtistList: View {
     let artists: [Artist]
     
+    var count = 0
+    var loadMore: LoadCallback = nil
+    
+    @State private var working = false
+    @State private var search: String = ""
+    
     var body: some View {
+        let filtered = filter(artists: artists)
```

Comment:
Isn't this function redundant when the callback is responsible for filtering?

Reply:
It could be, but I'm seeing the Tracks doing the same so I just did the same.

Reply:
I removed this already and will move search logic soon.

Reply:
Then I'm going to do the same for artists. Or did you already modified artists?

Reply:
No, I did not make any changes that would affect this pr